vector/src/main/res/layout/item_room.xml

```diff
@@ -84,6 +84,21 @@
         app:layout_constraintCircleAngle="135"
         app:layout_constraintCircleRadius="28dp"
         tools:ignore="MissingConstraints"
+        tools:visibility="gone" />
+
+    <ImageView
+        android:id="@+id/roomAvatarPresenceImageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:background="@drawable/background_circle"
+        android:contentDescription="@string/a11y_public_room"
```

Comment:
bad

Reply:
Can you be more specific? 
 - `wrap_content` is replaced by static `16dp`
 - `android:contentDescription` needs translations

Reply:
Yes, sorry, `a11y_public_room` is not the correct value to use here.